examples/objectDetection-webcam/sketch.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates detecting objects in a live video through ml5.imageClassifier.
+ */
+
+let video;
+let detector;
+let detections = [];
+
+function preload(){
+  detector = ml5.objectDetector("cocossd");
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // Using webcam feed as video input, hiding html element to avoid duplicate with canvas
+  video = createCapture(VIDEO);
+  video.size(width, height);
```

Comment:
Ah, I just tested and confirm that the #264 applies here too! If I change the webcam to `320,240` then the bounding boxes are all off. @yiyujin I don't think this has to be fixed in this PR, it can be done separately after the fact or along with #264. @nasif-co maybe you can help track this in the issue thread or a new issue so that we can merge at least the SelfieSegmentation fix?